backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientMutationResolver.java

```diff
@@ -83,6 +85,8 @@ public Person addPatient(
             ? phoneNumbers
             : List.of(new PhoneNumberInput(null, parsePhoneNumber(telephone)));
 
+    var backwardsCompatibleEmails = new PatientEmailsHolder(email, emails);
```

Comment:
Same thoughts on my confusion of what backwardsCompatibleEmails means. See comment below

Reply:
See https://github.com/CDCgov/prime-simplereport/pull/2897#discussion_r743999626